### DISCUSSION TITLE
Is flet ready for soemthing like a discussion forum like app

### DISCUSSION THREAD

**Discussion by saisaigraph** [flet-dev/flet#653] [General]
Just wondering Is flet ready for soemthing like a discussion forum like app or is it in too early stage.
Any thoughts or suggestions please.
Thanks
Sai

**Comment by FeodorFitsner** [comment]
I'm pretty sure it's ready!

Disclaimer: I created this framework, so can be biased :)

**Comment by ndonkoHenri** [comment]
By the way, we are still waiting to See the Chat App @FeodorFitsner 🙂
